js/index.js

```diff
@@ -13,4 +13,26 @@ for (const item of skills){
     skill.innerText = item;
     skillsList.appendChild(skill);
 }
-    
+let messageSection = document.getElementById('messages');
+let messageList = messageSection.querySelector('ul');
+let newMessage = document.createElement('li');
+newMessage.getElementByName('usersName').innerHTML = "<a href=mailto: + newMessage.getElementByName('usersEmail').email>";
+removeButton.innerText = 'remove';
+removeButton.Type = 'button';
+removeButton.addEventListener('click', () => {
+    let entry = document.getElementByType('button').parentNode.nodeName;
+    entry.remove();
+    newMessage.appendChild(removeButton);
+    messageList.appendChild(newMessage);
+});
+messageForm.addEventListener("submit", function(event){
+document.getElementsByName('leave_message').reset();    
+event.preventDefault();
+    let usersName = event.target.usersName;
+    let usersEmail = event.target.usersEmail;
+    let usersMessage = event.target.usersMessage;
+    console.log(usersName);
+    console.log(usersEmail);
+    console.log(usersMessage);
+ }, true);   
+const messageForm = document.getElementsByName('leave_message');
```

Comment:
This is after the use of messageForm and returns an array.  Also better to use an id and use getElementById which will return just one element.  Change us an id attribute instead of a name attribute on the form element.